modules/administration-guide/partials/proc_configuring_bitbucket_authentication.adoc

```diff
@@ -0,0 +1,52 @@
+// configuring-bitbucket-authentication
+
+[id="configuring_bitbucket_authentication_{context}"]
+= Authentication on Bitbucket servers
+
+{prod} users may use public or private Bitbucket Git repositories as a source of theirs projects. The standard
+factory flow using devfile at the root of the repository is available starting of {version} of {prod}.
+
+Using private repositories, requires some additional configuration described below.
+
+Bitbucket authentication is based on using so-called "personal access tokens". Each Bitbucket user is able to request some
+amount of personal access tokens with different names, permissions, expiration times and so on. Those tokens
+may be used to sign Bitbucket REST API calls and perform Git repository operations.
+
+To allow Bitbucket authentication on {prod} side, personal tokens must be stored in user's namespace in a form of
+secret. The secret must look as follows:
+
+[source,yaml]
+----
+include::example$snip_bitbucket-personal-access-token-secret.adoc[]
+----
+
+The main parts of the secret are:
+
+[cols=2*]
+|===
+| `app.kubernetes.io/component`
+| Label indicates it is a SCM personal token secret
+
+| `che.eclipse.org/che-userid`
+| {prod} id of the user token belongs to
+
+| `che.eclipse.org/scm-userid`
+| SCM id of the user token belongs to
+
+| `che.eclipse.org/scm-username`
```

Comment:
Are all of these parts labels except `token` or only `app.kubernetes.io/component` while other are expected as secret items?

Reply:
Ah, from the example above I got that they are annotations, but I think this table should be self-sufficient

Reply:
I added the type column